pkg/chaosdaemon/httpchaos_server.go

```diff
@@ -131,10 +143,11 @@ func (s *DaemonServer) applyHttpChaos(ctx context.Context, logger logr.Logger, i
 	}
 
 	return &pb.ApplyHttpChaosResponse{
-		Instance:   int64(in.Instance),
-		StartTime:  in.StartTime,
-		StatusCode: int32(resp.StatusCode),
-		Error:      string(body),
+		Instance:    int64(in.Instance),
+		InstanceUid: in.InstanceUid,
```

Comment:
It seems this Uid will be ignore?

Reply:
We mat not need to return it.

Reply:
Currently, the controller-manager does not use this UID, I would like to check the compatibility in this PR. I will create another PR to use UID instead of ProcessPair.

Reply:
OK.